dts/arm/nxp/nxp_rt7xx_cm33_cpu0.dtsi

```diff
@@ -1201,6 +1201,18 @@
 		ulps-control;
 		status = "disabled";
 	};
+
+	mbox_hifi4: mbox@189000 {
+		#mbox-cells = <1>;
```

Comment:
I think we can call this mbox4 just to be consistent with the rest of the dtses reference for mbox nodes.